### DISCUSSION TITLE
NRF51 SDK 8.0.0

### DISCUSSION THREAD

**Discussion by sabulaban** [bitcraze/discussions#423] [Q&A] [answered]
Anyone tried building the nrf firmware using SDK 8.0.0? SDK 8.0.0 supports nrf51822

**Comment by ataffanel** [comment, marked as answer]
No, to my knowledge there has been no attempt to update the SDK. Do you have any specific use-case for using an updated version? 

Since we are not using much of the SDK code, the only major advantage will be an updated BLE stack. For the rest of the nRF code I have been contemplating using mode the [nrfx](https://github.com/NordicSemiconductor/nrfx) open-source lib by Nordic but has never got around to do it.

**Reply by sabulaban** [reply]
Thank you for your response...the reason I want to update the firmware is to benefit from BLE_GAP_EVT_ADV_REPORT which is not available in 7.3.0, and the idea is on an adv event, I'll encapsulate the adv into CRTP port 13, channel 3, for use by an app.
